src/codeu/chat/client/simplegui2/ChatMenu.java

```diff
@@ -298,8 +299,12 @@ private void getAllMessages(ConversationSummary conversation) {
       // Display author name if available.  Otherwise display the author UUID.
       final String authorName = clientContext.user.getName(m.author);
 
+      Time time= m.creation;
+      String cutSecondsOffTime= time.toString();
+      cutSecondsOffTime=cutSecondsOffTime.substring(0, cutSecondsOffTime.length()- 7);
```

Comment:
Interesting. However, the string might not always be the same length. For example, what if it was 0 milliseconds. Would it display 000 or 0? We should check on that first.